js.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function daysInYear(year) {
+  if (Number.isInteger(year)) {
+      let feb = new Date(year, 1, 29).getDate();
+      if (feb === 29) {
+          return console.log(366);
```

Comment:
console.log - return undefined. So your code is equivalent to `return undefined`.